lib/static/new-ui/features/suites/components/SnapshotsPlayer/index.tsx

```diff
@@ -12,16 +12,33 @@ import {getCurrentResult} from '@/static/new-ui/features/suites/selectors';
 import {Timeline} from './Timeline';
 import {NumberedSnapshot} from './types';
 import {loadSnapshotsFromZip, useLiveSnapshotsStream, useScaleToFit} from './utils';
-
-import '@rrweb/replay/dist/style.css';
-import styles from './index.module.css';
 import {getTestSteps} from '@/static/new-ui/features/suites/components/TestSteps/selectors';
 import {setCurrentHighlightStep, setCurrentStep} from '@/static/modules/actions';
 import {Step, StepType} from '@/static/new-ui/features/suites/components/TestSteps/types';
 import {unstable_ListTreeItemType as ListTreeItemType} from '@gravity-ui/uikit/unstable';
 import {useAnalytics} from '@/static/new-ui/hooks/useAnalytics';
 import BrokenSnapshotIcon from '@/static/icons/broken-snapshot.svg';
 import {PlayIcon} from './PlayIcon';
+import {ChangedDot} from '../../../../components/ChangedDot';
+
+import '@rrweb/replay/dist/style.css';
+import styles from './index.module.css';
+
+const SETTINGS_OPTIONS: SelectOptions = [
+    {
+        label: 'Speed',
+        data: {
+            hint: <span><b style={{fontWeight: 600}}>Note:</b> transitions or animations will still play at original speed.</span>
+        },
+        options: [
+            {value: '0.5', content: '0.5x'},
+            {value: '1', content: '1x'},
+            {value: '1.5', content: '1.5x'},
+            {value: '2', content: '2x'},
+            {value: '4', content: '4x'}
```

Comment:
to my mind we can stop at 2x (like in most services, youtube for example). If the test lasts longer than a minute then we can just move to the desired part of the test

Reply:
I think it won't harm to have it here tbh. I myself often need more than 2x on youtube